dockerfiles/README.md

```diff
@@ -10,6 +10,117 @@ There are also images for ARM32 and ARM64 architectures but no debug versions of
 
 For a detailed list of installation, usage and versions available, please refer to the the official documentation: https://docs.fluentbit.io/manual/installation/docker
 
-## Building
+## Multiple architecture support
 
-All container images for the `1.8` branch are built from the separate `fluent/fluent-bit-docker-image` repository on the `1.8` branch: https://github.com/fluent/fluent-bit-docker-image/tree/1.8
```

Comment:
I think we can't just pull the readme from master as for 1.8 container releases, these currently come from this repository and the commands below will not work as there is no Linux container definition.

We need to keep this documentation here and just add the Windows section.

Reply:
Ack